packages/BaselineOfSandblocks/BaselineOfSandblocks.class.st

```diff
@@ -1,7 +1,7 @@
 Class {
 	#name : #BaselineOfSandblocks,
 	#superclass : #BaselineOf,
-	#category : 'BaselineOfSandblocks'
+	#category : #BaselineOfSandblocks
```

Comment:
Huh? This was Squot, not me 🤔

Reply:
looks like it may be a newer convention. I just checked and saw it formatted as symbol in another more recent project as well.